explore/src/main/webapp/main.jsx

```diff
@@ -39,6 +39,5 @@ if (import.meta.hot) {
   import.meta.hot.accept();
   import.meta.hot.dispose((_) => {
```

Comment:
Do we need this at all if it doesn't do anything?

Reply:
We don't really need it I thought to keep it as a reminder but maybe I should rather delete it